src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
+$secondary-color-gray: #616070;
+$buttun-color: #00acdc;
+$margin-center: 0 auto;
+$family-roboto: "Roboto", "Helvetica", sans-serif;
```

Comment:
you can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`)